### PR TITLE
[!!!][BUGFIX] synchronize extension dependencies between "composer.json" and "ext_emconf.php"

### PR DESCRIPTION
**Pull request by websi**
Move non direct dependencies to section "suggest"

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -26,28 +26,26 @@
...
-    "svewap/realurl-404-multilingual": "1.0.9",
-    "typo3-ter/frontend-editing": "1.1.2"
-    }
+    "typo3-ter/realurl": "~2.2.1",
+    "clickstorm/go_maps_ext": "~2.3.0",
+    "clickstorm/cs_seo": "~2.1.2",
+    "typo3-ter/news": "~6.0.0",
+    "typo3-ter/frontend-editing": "~1.1.2"
+  },
+  "suggest": {
```
Comment (dmh) [inline composer.json:44]: In general, looks good, except I am not sure about `suggest` section. All those extensions are a core part of t3kit, so it is not just suggestion, it is integrated into t3kit deeper (ts configs, templates, etc...)
Thanks! 
@pixelmatseriks @MattiasNilsson ?

**Review by dmh — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (websi) [reply composer.json:44]: Ok. That's right for `solr` and `pxa_newsletter_subscription` . For the any other extension, i can not found any source code for that extensions.

Comment (dmh) [reply composer.json:44]: Those extensions included also in our base t3kit db
https://github.com/t3kit/t3kit_db

Comment (websi) [reply composer.json:44]: Yes, you write it.
These extensions are no dependencies to `theme_t3kit` extension.
They are dependencies for a t3kit project with your databse content, but not for the extension itself.

At a closer look, the extension `cs_seo` should also moved to section "suggest".
The configration for that extension are only loaded, if extension is installed.

Comment (dmh) [reply composer.json:44]: It sounds reasonable to me. We will move those extensions as the main dependencies to `composer.json` in `t3kit` repo, and then we can use it as a suggestion here in `theme_t3kit`.

So would be nice if you fix conflicts and update your PR.

Thanks!

Comment (websi) [reply composer.json:44]: Ok, I will move `solr` and `pxa_newsletter_subscription` back to section "required".
To analyse which extension is only needed for an optional feature of the `theme_t3kit` extension, I will open an issue. After this analyse, I will open an new pull request for these extension, if needed.

**Pull request closed without merging**